src/container/srv_container.c

```diff
@@ -3139,6 +3139,10 @@
 		struct cont_query_out  *cqo = crt_reply_get(rpc);
 
 		prop = cqo->cqo_prop;
+	} else if (opc == CONT_OPEN) {
+		struct cont_open_out *coo = crt_reply_get(rpc);
```

Comment:
```suggestion
		struct cont_open_out *coup = crt_reply_get(rpc);
```

Reply:
you might consider adding coo to the list of words that are acceptable to avoid future problems with spell check

ci/codespell.ignores